framework/doc/content/source/meshgenerators/MergedMeshGenerator.md

```diff
@@ -0,0 +1,16 @@
+# MergedMeshGenerator
+
+## Overview
+
+The `MergedMeshGenerator` object allows for multiple mesh files to be "merged"
+together to form a single mesh for use in a simulation. This generator will keep
+all submeshes disconnected from each other. Interaction between submeshes has to
```

Comment:
Since this could be used for other physics, I'd make the last sentence more general.  Maybe 'Interaction between submeshes only occurs due to contact or other constraints'.

Reply:
Also, doesn't the name "Merged" sort of imply that the meshes are made to connect with each other? In Cubit, the merge command merges two adjacent bodies together to ensure that the mesh is contiguous.

Reply:
Yeah, I agree. Any suggestions for a better name?

Reply:
I was brainstorming ideas on the way home yesterday.  Here are a few I came up with:
OverlaidMeshGenerator, UnionMeshGenerator, CombineMeshGenerator

Reply:
Overlaid implies that the meshes are on top of each other, Union and Combine are not better than Merge in my opinion.

Reply:
CollectionMeshGenerator?

Reply:
I'm OK with merged, but merge to me implies some sort of "joining", perhaps "stitching". I actually like Ben's suggestion of "UnionMeshGenerator" in terms of set theory. Unions bring together disjoint sets into one, which I personally find clear.

Reply:
This has already been renamed to `MeshCollectionGenerator`